src/pytorch_lightning/strategies/ipu.py

```diff
@@ -42,25 +42,8 @@
 
 
 class LightningIPUModule(_LightningModuleWrapperBase):
```

Comment:
we can remove this? since it doesn't do anything now.

Reply:
If the logic here is really not needed, I'm also in favor (removal with deprecation)

Reply:
I have removed `LightningIPUModule`, but I'm not sure what you're deprecation strategy is so please let me know if there's anything else I need to do